guides-and-tutorials/sbtc/how-to-run-sbtc-signer.md

```diff
@@ -0,0 +1,531 @@
+{% endhint %}
+
+## Minimum System Requirements
+
+These are the **minimum required specs** to be able to run a sBTC signer, but it is recommended to have more than the minimum for optimal performance. Note that these are in addition to the hardware requirements for running a Stacks node, Bitcoin node, and Nakamoto signer outlined in the How to Run a Signer doc.
+
+- 1 cpu
+- 256MB memory
+- 50GB storage
+
+## 1. Configure your Bitcoin node
+
+Your Bitcoin node must include these settings for sBTC signer operation:
+
+1. Required Settings:
+
+   - `txindex=1`: Transaction indexing must be enabled
+   - `server=1`: RPC server must be enabled
+   - `zmqpubhashblock=tcp://*:28332`: ZMQ block hash notifications
+   - `zmqpubrawblock=tcp://*:28332`: ZMQ raw block notifications
+
+2. Optional but Recommended:
+   - `coinstatsindex=1`: Enables additional index for coin statistics
+
+Reference configuration:
+
+<details>
+
+<summary>bitcoin.conf example</summary>
+
+```conf
+regtest=1 #chain=regtest
+
+[regtest]
+printtoconsole=1
+disablewallet=0
+txindex=1
+coinstatsindex=1
+
+# Specify a non-default location to store blockchain data.
+# blocksdir=/data/bitcoin-data
+# Specify a non-default location to store blockchain and other data.
+# datadir=/data/bitcoin-data
+
+# [network]
+# bind=0.0.0.0:18444
+discover=0
+dns=0
+dnsseed=0
+listenonion=0
+
+# [rpc]
+rpcserialversion=0
+# Accept command line and JSON-RPC commands.
+server=1
+# Accept public REST requests.
+rest=1
+rpcbind=0.0.0.0:18333
+rpcallowip=0.0.0.0/0
+rpcallowip=::/0
+rpcuser=user
+rpcpassword=password
+
+# [zmq]
+# Note that this is required for the sbtc signer to work properly.
+zmqpubhashblock=tcp://*:28332
+zmqpubrawblock=tcp://*:28332
+
+# [wallet]
+addresstype=legacy
+changetype=legacy
+fallbackfee=0.00001
+```
+
+</details>
+
+### ZeroMQ (ZMQ) Configuration
+
+ZeroMQ enables real-time blockchain event notifications from Bitcoin Core to the sBTC signer. The two required ZMQ endpoints serve distinct purposes:
+
+- `zmqpubhashblock`: Broadcasts only block hashes for lightweight block detection
+- `zmqpubrawblock`: Broadcasts complete block data for transaction processing
+
+This notification system creates a direct event stream:
+
+1. Bitcoin Core validates a new block
+2. Block data publishes via ZMQ
+3. Signer processes relevant sBTC transactions
+
+## 2. Configure your signer
+
+The signer configuration file (`signer-config.toml`) defines the signer's operation parameters. The configuration sections include:
+
+### Emily API Configuration
+
+Controls connection to the Emily API server. The Emily API is designed to track deposits and withdrawals, providing information about the status of in-flight sBTC operations. It serves two primary user groups: sBTC users and sBTC app developers.
+
+```toml
+[emily]
+endpoints = ["https://emily-sbtc.com"]
+```
```

Comment:
Wouldn't this be a default value? Why would signers ever want to change this?

CC @AshtonStephens

Reply:
Yes, they would not change this and it would be a default value.